vertical-pod-autoscaler/pkg/admission-controller/config_test.go

```diff
@@ -244,3 +245,67 @@ func TestSelfRegistrationWithOutFailurePolicy(t *testing.T) {
 	assert.NotNil(t, *webhook.FailurePolicy, "expected namespace selector not to be nil")
 	assert.Equal(t, *webhook.FailurePolicy, admissionregistration.Ignore, "expected failurePolicy to be Ignore")
 }
+
+func TestConvertLabelsToMap(t *testing.T) {
+	testCases := []struct {
+		desc           string
+		labels         string
+		expectedOutput map[string]string
+		expectedError  bool
+	}{
+		{
+			desc:           "should return empty map when tag is empty",
+			labels:         "",
+			expectedOutput: map[string]string{},
+			expectedError:  false,
+		},
+		{
+			desc:   "sing valid tag should be converted",
+			labels: "key:value",
+			expectedOutput: map[string]string{
+				"key": "value",
+			},
+			expectedError: false,
+		},
+		{
+			desc:   "multiple valid labels should be converted",
+			labels: "key1:value1,key2:value2",
+			expectedOutput: map[string]string{
+				"key1": "value1",
+				"key2": "value2",
+			},
+			expectedError: false,
+		},
+		{
+			desc:   "whitespaces should be trimmed",
```

Comment:
what about adding a test with whitespace between the keys and values?
`key1 : value1,key2 : value2`

Reply:
done